testing-common/src/main/groovy/io/opentelemetry/auto/test/AgentTestRunner.java

```diff
@@ -68,17 +68,17 @@
  * <ul>
  *   <li>All {@link Instrumenter}s on the test classpath will be applied. Matching preloaded classes
  *       will be retransformed.
- *   <li>{@link AgentTestRunner#TEST_WRITER} will be registerd with the global tracer and available
+ *   <li>{@link AgentTestRunner#TEST_WRITER} will be registered with the global tracer and available
  *       in an initialized state.
  * </ul>
  */
+@Category(BytecodeTests.class)
 @RunWith(SpockRunner.class)
 @SpecMetadata(filename = "AgentTestRunner.java", line = 0)
 public abstract class AgentTestRunner extends AgentSpecification {
 
   private static final org.slf4j.Logger log = LoggerFactory.getLogger(AgentTestRunner.class);
 
-  private static final long TIMEOUT_MILLIS = 10 * 1000;
```

Comment:
Yes! I tried to remove this before but ended up with totally inexplicable test failures even though it's not used anywhere